train/train_params.py

```diff
@@ -109,4 +103,8 @@ def make_tp(forgiving=False, takes_extras=False, convert_lists_to_tuples=False,
 #   Relevant only when also passing --save through the args, otherwise will be
 #   overridden to 0.
 # early_stop_nsamples:
-#   When >0, the number of batches to train before early stopping the training.
\ No newline at end of file
+#   When >0, the number of batches to train before early stopping the training.
+# weight_decay:
+#   Weight decay to apply to the optimizer.
+# random_seed:
+#   Random seed for reproducibility
```

Comment:
maybe add a note here that this value can also be overwritten by setting the random seed arg in main